src/foam/u2/crunch/WizardRunner.js

```diff
@@ -76,12 +79,17 @@ foam.CLASS({
             });
           });
         }
-
-        await this.crunchController.inlineWizardFromSequence(parentWizard, seq, ( returnPromise ? { onLastWizardletSaved: () => promise$.set(true) } : {}));
-        return returnPromise ? returnPromise : null;
+        wizardContext = await this.crunchController.inlineWizardFromSequence(
+          parentWizard,
+          seq, 
+          (options.returnCompletionPromise ? { onLastWizardletSaved: () => promise$.set(true) } : {})
+        );
+        await returnPromise;
+      } else {
+        wizardContext = await seq.execute();
       }
 
-      await seq.execute();
+      return wizardContext;
```

Comment:
Where do we use this?

Reply:
Just saw the NP PR, have some thoughts on how to do this, sine each wizardRunner instance is a unique representation of a wizard anyway, it could store the wizard it launches in a property that can then be subbed to/accessed